bin/trivy_validator.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+import json
+import os
+from pathlib import Path
+
+git_root = [x for x in Path(__file__).resolve().parents if (x / ".git").is_dir()][-1]
+project_name = os.getenv("PROJECT_DIRECTORY")
+scan_results_file_path = f"{git_root}/{project_name}"
+
+data = json.loads(Path(f"{scan_results_file_path}/scan-results.json").read_text())
+found_cves = [vuln["VulnerabilityID"] for result in data.get("Results", []) for vuln in result.get("Vulnerabilities", [])]
+
+
+def cve_list_to_string(items: list):
+    sorted_items = sorted(set(items), key=lambda x: x.lower() if x else "")
+    if sorted_items:
+        return "\n\n" + "\n".join([f"{item}" for item in sorted_items]) + "\n"
+    return "none\n"
+
+
+def get_ignored_cves() -> list:
+    ignore_file = Path(f"{scan_results_file_path}/trivyignore")
+    if ignore_file.is_file():
+        return sorted({line.strip() for line in ignore_file.read_text().splitlines()})
+    return []
```

Comment:
i can import this same function from twist_cli validator python script since both share the same code

Reply:
I don't think doing the code changes necessary to make that work are important. It's more important that this script be self contained.